packages/manager/cypress/e2e/databases/update-database.spec.ts

```diff
@@ -247,7 +251,7 @@ describe('Update database clusters', () => {
    * - Confirms that users cannot reset root passwords for provisioning DBs.
    * - Confirms that users cannot change maintenance schedules for provisioning DBs.
    */
-  it('Cannot update database clusters while they are provisioning', () => {
+  it.skip('Cannot update database clusters while they are provisioning', () => {
```

Comment:
```suggestion
  it('Cannot update database clusters while they are provisioning', () => {
```
Ah sorry about this, I forgot to remind you to delete the `.skip` once we finished development! Applying this suggestion should take care of it.

Otherwise looking good!